### PR TITLE
TFAC-1165 Preliminary Experiment Setup

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: graphql/database/experiments/features.js [hunk 1]
```diff
@@ -328,6 +329,59 @@ const features = {
...
+      //   coverage: 1.0,
+      //   condition: {
+      //     v4BetaEnabled: {
+      //       $eq: true,
+      //     },
+      //     tabs: {
+      //       $gt: 3,
+      //     },
+      //     joined: {
+      //       $gt: SFAC_EXTENSION_PROMPT_CUTOFF_UNIX_TIME,
```
Comment (kmjennison) [inline graphql/database/experiments/features.js:379]: This time filter won't be necessary.

**Review by kmjennison — APPROVED** [review]
LGTM ☕

File: graphql/database/experiments/features.js [hunk 2]
```diff
@@ -328,6 +329,59 @@ const features = {
...
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
+          },
+          tabs: {
+            $gt: 3,
+          },
+          [`internalExperimentOverrides.${SEARCHBAR_SFAC_EXTENSION_PROMPT}`]: {
+            $eq: 'Control',
```
[the remark this answers is not in the document]
Comment (jedtan) [reply graphql/database/experiments/features.js:345]: Yep! Also, just to clarify, the other experiment is actually already ended right? If you look at the `SFAC_EXTENSION_PROMPT` feature def, there's no more experiment.

[the remark this answers is not in the document]
Comment (kmjennison) [reply graphql/database/experiments/features.js:345]: Right, sorry! Wrote this, saw that, forgot to edit my comment.

**Pull request merged**